src_kt/java/arcs/crdt/CrdtSet.kt

```diff
@@ -18,10 +18,13 @@ import arcs.crdt.internal.VersionMap
 
 /** A [CrdtModel] capable of managing a set of items [T]. */
 class CrdtSet<T : Referencable>(
-  private var _data: Data<T> = Data()
+  /** Initial data. */
+  private var _data: Data<T> = DataImpl(),
```

Comment:
Is prefixing private variables with _ a Kotlin style thing, or is this a holdover from the Typescript?

Reply:
it's a thing when they're backing-values for a property with the non-underscored version of the name.

Reply:
[Context:](https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules) "Names for backing properties."